ts/src/renderer/phaser/classes/PhaserProjectile.ts

```diff
@@ -22,27 +24,32 @@ class PhaserProjectile extends Phaser.GameObjects.Container {
 				console.log('PhaserProjectile play-animation', `${key}/${animationId}`);  // TODO remove
 				sprite.play(`${key}/${animationId}`);
 			});
+
+		//console.log('projectile', projectile._scale, projectile._pixiTexture.scale, projectile._stats.currentBody.width)
 	}
 
 	update (/*time: number, delta: number*/): void {
 
 		const projectile = this.projectile;
-		const container = projectile._pixiContainer;
-		const texture = projectile._pixiTexture;
+		//const container = projectile._pixiContainer;
+		//const texture = projectile._pixiTexture;
 
-		if (projectile._destroyed || container._destroyed) {
+		if (!projectile._alive/*projectile._destroyed || container._destroyed*/) {
```

Comment:
`_alive` seems like a good alternative to `__destroyed`

Reply:
i think the intention is `._destroys` indicates it is marked for clean-up

Reply:
good point, might be smart to keep track of `_destroyed` in ige entity, same as is now for pixi object, and query that instead